### DISCUSSION TITLE
Centered sight

### DISCUSSION THREAD

**Discussion by L4nzinhu** [redphx/better-xcloud#193] [General]
Well, I found this initiative on your part to create an optimizing script for "Xcloud" very interesting, however I noticed that the option to use keyboard and mouse was added, and within that option I saw that the mouse was much superior to other extensions that I've seen before, like this, I believe that if it were possible to leave the mouse centered on the screen (it doesn't change its position when I move it within the game, it just changes the game's camera) it would make it much better to play.
Furthermore, I also believe that you can solve the problem of having to rotate the mouse to the extreme for the camera to rotate.

This is just constructive criticism, I don't want to interfere with your time :)

**Comment by redphx** [comment]
I just updated the mouse controlling feature. Not sure if it's still what you wanted or not.

**Reply by L4nzinhu** [reply]
I'll try it out, thanks for your work and happy new year :)

**Reply by L4nzinhu** [reply]
Well, anyway it's good, now I think it would be better to make the mouse invisible even when they are not stationary.
:)

**Reply by L4nzinhu** [reply]
![image](https://github.com/redphx/better-xcloud/assets/155100711/1d39d1e1-7b09-4025-9a77-4ffe7082b461)
not work :(

**Reply by redphx** [reply]
try to install the script again

**Reply by L4nzinhu** [reply]
this work now :D

**Reply by redphx** [reply]
If you found any problem please report it in the main issue page
